supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -0,0 +1,69 @@
+package cromwell.backend.impl.jes.statuspolling
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import cats.data.NonEmptyList
+import cromwell.backend.impl.jes.Run
+import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager._
+
+/**
+  * Currently, just holds a set of JES status poll requests until a PollingActor pulls the work.
+  * TODO: Could eventually move all of the JES queries into a single work-pulling model.
+  */
+class JesApiQueryManager extends Actor with ActorLogging {
+
+  private var workQueue: List[JesStatusPollQuery] = List.empty
+
+  // TODO: Random magic number 5. Dynamically resize this?
+  // TODO: Supervision strategy for pollers should be: stop that one, tell its requesters that it failed, and create a new one to replace it.
+  private val pollers = (0 until 5) map { i => context.actorOf(JesPollingActor.props(self)) }
+  pollers foreach { _ ! NoWorkToDo }
+
+  override def receive = {
+    case DoPoll(run) => workQueue :+= JesStatusPollQuery(sender, run)
```

Comment:
Appends to a `List` are O(n); this should probably be a different data structure.